RelayProofsOfConcept/EddsaTokioServer/examples/eddsa_key_gen_client.rs

```diff
@@ -327,7 +245,7 @@ impl Peer for EddsaPeer {
         let res = fs::write(env::args().nth(2).unwrap(), keygen_json);
         match res {
             Ok(_) => Ok(()),
-            Err(e) => Err("Failed to verify"),
+            Err(_e) => Err("Failed to verify"),
```

Comment:
you can also just write `Err(_)` in this case.

Reply:
Fixed